src/containers/Penjualan/Table/TablePenjualan.tsx

```diff
@@ -28,10 +29,21 @@ type handleShowType = {
 
 const TablePenjualan = (props: handleShowType) => {
   const { showAdd, showEdit, setShowAdd, setShowEdit } = props
-  const [cookies] = useCookies(['token', 'user'])
+  const { user } = useMarContext()
+  const [cookies] = useCookies(['token'])
```

Comment:
why need cookie? save token also in contex. better, when login, save default header token to axios. so you dont have to send header for each request. 
Read:
https://axios-http.com/docs/config_defaults

Reply:
done create default header token to axios